.editorconfig

```diff
@@ -0,0 +1,17 @@
+root = true
```

Comment:
Can you add a comment what this file is (with a pointer to https://editorconfig.org/), assuming .editorconfig allows putting comments in it?

Reply:
sure.